src/cli/tls_proxy.cpp

```diff
@@ -273,20 +286,21 @@ class tls_proxy_session : public boost::enable_shared_from_this<tls_proxy_sessio
          m_hostname = session.server_info().hostname();
 
          if(m_hostname != "")
+            {
             std::cout << "Client requested hostname '" << m_hostname << "'" << std::endl;
+            }
 
          async_connect(m_server_socket, m_server_endpoints,
                        [this](boost::system::error_code ec, tcp::resolver::iterator endpoint)
-                       {
-                       if(ec)
-                          {
-                          log_error("Server connection", ec);
-                          return;
-                          }
-
-                       server_read(boost::system::error_code(), 0); // start read loop
-                       proxy_write_to_server(nullptr, 0);
-                       });
+            {
```

Comment:
This indentation seems wrong. Might be better to first define the lambda in a variable, then pass it to `async_connect`.